src/data/localStorage/ParticipantCode.ts

```diff
@@ -1,13 +1,44 @@
 import AsyncStorage from '@react-native-async-storage/async-storage';
 
+export interface questionnaire {
+  code: string;
+  name: string;
+}
+
 class ParticipantCode {
-  public static async saveParticipantCodeToLocalStorage(code: string) {
+  public static async saveCurrentParticipantCodeToLocalStorage(code: string) {
     await AsyncStorage.setItem('ParticipantCode', code);
   }
 
-  public static async loadParticipantCodeFromLocalStorage() {
+  public static async loadCurrentParticipantCodeFromLocalStorage() {
     return await AsyncStorage.getItem('ParticipantCode');
   }
+
+  private static codeAppearsInQuestionnaires(questionnaires: questionnaire[], code: string) {
+    let codeAppears = false;
+    questionnaires.forEach((q) => {
+      if (q.code === code) {
+        codeAppears = true;
```

Comment:
Waarom zet je de waarde naar true? Je kan toch gewoon returnen?

Reply:
Je kan niet uit een foreach stappen in javascript, dan zal het altijd false returnen. Je kan wel every gebruiken, daar stap je uit de loop als je false returnt.

Reply:
Aha